qubes-vm/qrexec/qrexec-4.0.39.ebuild

```diff
@@ -51,6 +48,7 @@ src_install() {
 	default
 	dotmpfiles "${FILESDIR}"/qubes-vm-qrexec.conf
 	install_systemd_units "${SYSTEMD_UNITS[@]}"
+	newinitd vm-init.d/qubes-qrexec-agent
```

Comment:
let's copy function from core-agent and use that for consistency. Later we will go through common functionality and put that together in an eclass or sth.

Reply:
isn't this missing second arg? or should it actually be `doinitd`?